testdata/scripts/embed.txt

```diff
@@ -0,0 +1,32 @@
+env GOPRIVATE=*
+
+garble build
+
+exec ./main
+
+cmp stderr main.stderr
+
+-- go.mod --
+module test/main
+
+go 1.16
+
+-- main.go --
+package main
+
+import (
+	_ "embed"
+)
+
+//go:embed test.txt
+var x string
+
+func main() {
+	println(x)
+}
+
+-- test.txt --
+test
```

Comment:
you could make this `test content`, to make it clearer that it's unrelated to the filename